batch.R

```diff
@@ -158,15 +158,15 @@ for(i in 1:nrow(fileDF)) {
                                       rloadest5param = rloadest5param)
   
   #make predictions
-  pred_rload <- predictSolute(rloadest5param, "flux", siteQ, 
-                              se.pred = TRUE, date = TRUE)
-  pred_interp <- predictSolute(interpRect, "flux", siteQ, 
-                               se.pred = TRUE, date = TRUE)
-  pred_comp <- predictSolute(comp, "flux", siteQ, se.pred = TRUE,
-                             date = TRUE)
+  pconc_rload <- predictSolute(rloadest5param, "conc", siteQ, se.pred = TRUE, date = TRUE)
+  pconc_interp <- predictSolute(interpRect, "conc", siteQ, se.pred = TRUE, date = TRUE)
+  pconc_comp <- predictSolute(comp, "conc", siteQ, se.pred = TRUE, date = TRUE)
+  pflux_rload <- predictSolute(rloadest5param, "flux", siteQ, se.pred = TRUE, date = TRUE)
+  pflux_interp <- predictSolute(interpRect, "flux", siteQ, se.pred = TRUE, date = TRUE)
+  pflux_comp <- predictSolute(comp, "flux", siteQ, se.pred = TRUE, date = TRUE)
   nPreds <- nrow(siteQ)
-  allPreds <- bind_rows(pred_rload, pred_interp, pred_comp)
-  allPreds$model <- c(rep("rloadest",nPreds), rep("interp",nPreds), rep("composite", nPreds))
+  allPreds <- bind_rows(pconc_rload, pconc_interp, pconc_comp)
+  allPreds$model <- rep(c('rloadest','interp','composite'), each=nPreds)
```

Comment:
I changed this line just in passing. Did you know about the `each` argument to `rep`? I like it for cases like this one.